lib/chartmogul/customer.rb

```diff
@@ -66,6 +66,17 @@ def self.merge!(into_uuid:, from_uuid:)
       true
     end
 
+    def self.unmerge!(customer_uuid:, data_source_uuid:, external_id:, move_to_new_customer: [])
+      options = {
+        customer_uuid: customer_uuid,
+        data_source_uuid: data_source_uuid,
+        external_id: external_id,
+        move_to_new_customer: move_to_new_customer
```

Comment:
[UX] would it make sense to save `["tasks", "opportunities", "notes"]` as some constant like `Customer.UNMERGE_MOVE_ALL`?